<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hierarchy</class>
 <widget class="QWidget" name="Hierarchy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="EntityList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddEntity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Entity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveEntity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Entity</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
